auth/src/lib.rs

```diff
@@ -35,6 +35,27 @@ pub mod login {
         pub server_data: String,
         pub credential_finalization: opaque::client::login::CredentialFinalization,
     }
+
+    #[derive(Serialize, Deserialize, Clone)]
+    pub struct ClientConventionalLoginRequest {
```

Comment:
Which convention? :)

Maybe better to rename to simple http login

Reply:
Changed it to ClientSimpleLoginRequest